src/serverlist.cpp

```diff
@@ -488,14 +537,25 @@ void CServerListManager::SlaveServerRegisterServer ( const bool bIsRegister )
         if ( bIsRegister )
         {
             // register server
+            eSvrRegStatus = SRS_REQUESTED;
+            emit SvrRegStatusChanged();
+
             pConnLessProtocol->CreateCLRegisterServerMes ( SlaveCurCentServerHostAddress,
                                                            SlaveCurLocalHostAddress,
                                                            ServerList[0] );
         }
         else
         {
             // unregister server
+            eSvrRegStatus = SRS_UNREGISTERED;
+            emit SvrRegStatusChanged();
+
             pConnLessProtocol->CreateCLUnregisterServerMes ( SlaveCurCentServerHostAddress );
         }
     }
+    else if ( bIsRegister )
```

Comment:
May be able to remove the `if` now.  (Was having strange problems at one point...)

Reply:
Do you have fixes in the queue or are you done with this?

Reply:
Still going to do these last few I commented on - they're not show stoppers, it works without them...  (It might work _better_ as is, indeed!  But some of it I changed when things were being a bit weird... so may be better removed.)

Not had much time over the last couple of days (Easter was better :) ).  So I'll be reviewing it all again, maybe trying a couple of things out.  What I'll then do is probably close off this PR, squash all the commits and raise a new one to tidy up.

Reply:
No rush, take your time. You have spend a lot of time already for Jamulus coding. Thank you very much for that!